spek-ide-plugin/intellij-common/src/main/kotlin/org/spekframework/intellij/SpekRunLineMarkerContributor.kt

```diff
@@ -0,0 +1,21 @@
+package org.spekframework.intellij
+
+import com.intellij.execution.lineMarker.ExecutorAction
+import com.intellij.execution.lineMarker.RunLineMarkerContributor
+import com.intellij.icons.AllIcons
+import com.intellij.psi.PsiElement
+import com.intellij.util.Function
+
+class SpekRunLineMarkerContributor: RunLineMarkerContributor() {
+
+    override fun getInfo(element: PsiElement): Info? {
+        val path = extractPath(element)
+        return path?.let {
+            Info(
+                AllIcons.RunConfigurations.TestState.Run,
+                Function<PsiElement, String> { "Run ${path.name}"},
+                *ExecutorAction.getActions(0)
```

Comment:
`0` seems like a magic value 🤔

Reply:
This is how most implementations write it, I can extract it into a constant variable but don't know what to call it.